legnardotmnt.yml

```diff
@@ -0,0 +1,27 @@
+class: leonardotmnt
+costumes:
+- chatcolor: '&1'
+  name: &2Leo&1nar&2do&8(&2TMNT&8)
+  sig: nyJN+e+QNIGydYedtvlnO8FxVE5LJ/mmA7Us2M839Cnt7h4M11QFEfD/6i5QCEGG7mLDjEJr3jGBLLgAaz3BkcwqORUKn2NRuzMlDM8mOavuc9yfaufIL9BNLD0YngZyTwTk5cNH1oQRUkeoGvw/8NmphFoHbKQeWO3EpHjZxClx8jaKNIgPP+qpkydtYbEcCLtnu86yNAN4hImjHJin2hqFa/7vJbd/COwTRx+GjrocKThZvodj3vbfzh+dVyE/oUhPj1NGjdpjHG9C5YdnwB7D4Nz7P8ThtdhtDZ3YnRM3PJ8Bu7p0w+y+CqYQge/GN1phyWW3rGlKX/bEyytgsj2z6lnRRyhRfSu+nJt3hnPONWQntsgQtcaJTrc9wkS+kvO8jCrgw4c+QW5Q467cdMozIFai/sDKnSX7hIeepZ/RN8lRTwRLqzlxUsV9ZjMjXbCp/8zI72WPPk0LmLrPqK3JuLDwvuM0MVDTSnp3zexnCH2ryymi99wQsDGwHwV0mE4IxY6eElfG2ghDcIa95OUYEkFOl6r7KujW1JNBh/ix3pkGohwUeIywaffoGV/gd0WV9sd8/D/jqOFEhTQCkmu3D9v1WDZuo8Rp0TyV0gkKu67ivYEwxTHDkG+Zd0Ug5yr30L1o29q/Efpc7Uhpvv1dZb3npUZeZzOgLJt9X70=
+  val: eyJ0aW1lc3RhbXAiOjE1MjMzMzEzNjIzNTMsInByb2ZpbGVJZCI6ImVmN2JjYzk2M2FjMDRkYzZhYWU3MDE5ZTVjNmRkYTJlIiwicHJvZmlsZU5hbWUiOiJzdXBlcmlvcnhtYW4iLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlLzg2NWYyMjE0YWMxMmQzODgzMjU1ODJiMmVmNWMzMDc1OTVhNTVkM2M1ZjhjYTUxODIyMWI0YzdhOGEyMzZjIn19fQ==
+effects:
```

Comment:
Since there are no effects please change it to this:

effects: [
  ]

Reply:
lightshadow is correct

you can also use

effects: []